src/Formatter/IntlMoneyFormatter.php

```diff
@@ -16,13 +16,19 @@
      * @var \NumberFormatter
      */
     private $formatter;
+    /**
+     * @var int|null
```

Comment:
Sorry, I missed this check:
`if ($this->subunits === null) {`

I thought there was no check for being set, so using null always casted to integer (0) seemed to be unnecessary.